src/algorithms/calorimetry/CalorimeterClusterMerger.cc

```diff
@@ -16,8 +16,13 @@ using namespace dd4hep;
 //------------------------
 // AlgorithmInit
 //------------------------
-void CalorimeterClusterMerger::AlgorithmInit() {
+void CalorimeterClusterMerger::AlgorithmInit(spdlog::level::level_enum loglevel) {
+    m_logger->set_level(loglevel);
```

Comment:
Log level should be set from Factory init ans is taken from users parameter. Example: 

```c++
// Get log level from user parameter or default
    std::string log_level_str = "info";
    pm->SetDefaultParameter("Tracking:RawHitReco:LogLevel", log_level_str, "verbosity: trace, debug, info, warn, err, critical, off");
    m_log->set_level(eicrecon::ParseLogLevel(log_level_str));
```

Also note this
https://github.com/eic/EICrecon/discussions/68

and review of logger init below